2007-04-01  Jakub Jelinek  <jakub@redhat.com>

	* sysdeps/ia64/fpu/fesetround.c (fesetround): Return 0 on success
	and 1 on failure.

--- libc/sysdeps/ia64/fpu/fesetround.c	8 Jul 2005 18:52:46 -0000	1.4
+++ libc/sysdeps/ia64/fpu/fesetround.c	1 Apr 2007 16:37:33 -0000	1.5
@@ -26,7 +26,7 @@ fesetround (int round)
   fenv_t fpsr;
 
   if (round & ~3)
-    return 0;
+    return 1;
 
   /* Get the current state.  */
   __asm__ __volatile__ ("mov.m %0=ar.fpsr" : "=r" (fpsr));
@@ -37,6 +37,6 @@ fesetround (int round)
   /* Put the new state in effect.  */
   __asm__ __volatile__ ("mov.m ar.fpsr=%0" :: "r" (fpsr) : "memory");
 
-  return 1;
+  return 0;
 }
 libm_hidden_def (fesetround)
